tasks/ThrusterLimitationTask.cpp

```diff
@@ -47,24 +47,42 @@ bool ThrusterLimitationTask::startHook()
     return true;
 }
 
-bool ThrusterLimitationTask::checkSpeedSaturation(commands::Joints const& cmd)
+bool ThrusterLimitationTask::checkEffortSaturation(commands::Joints const& cmd)
```

Comment:
Given that there is a JointLimits type ... I was wondering about having a method to check that a joint matches the limits.

https://docs.tidewise.io/cxx/structbase_1_1JointLimits.html

Reply:
Actually, if you felt like implemeting a `saturate` method on `JointLimits` ... that would make a whole lot of sense there as well.